src/clr-angular/popover/dropdown/dropdown-trigger.spec.ts

```diff
@@ -49,5 +52,25 @@ export default function(): void {
     it('declares itself to the DropdownFocusHandler', function(this: Context) {
       expect(this.getClarityProvider(DropdownFocusHandler).trigger).toBe(this.clarityElement);
     });
+
+    it('aria label should receive the correct value', function(this: Context) {
+      const ifOpenService = this.getProvider(IfOpenService);
+
+      ifOpenService.open = false;
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('Open dropdown');
+
+      this.hostComponent.openLabel = 'New value for open';
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('New value for open');
+
+      ifOpenService.open = true;
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('Close dropdown');
+
+      this.hostComponent.closeLabel = 'New value for close';
+      this.detectChanges();
+      expect(this.clarityElement.getAttribute('aria-label')).toBe('New value for close');
+    });
```

Comment:
Can we add a controller test that tests just the `ariaLabel` getter controller property that it returns the value based on if it is open or not?

Reply:
```   
      const ifOpenService = this.getProvider(IfOpenService);
      ifOpenService.open = false;
      this.detectChanges();
      expect(this.clarityDirective.ariaLabel).toBe(this.clarityDirective.clrAriaLabelDropdownOpen);
```
Like this but for both cases?

Reply:
Yes